ext/bcmath/bcmath.c

```diff
@@ -152,7 +152,11 @@ PHP_FUNCTION(bcadd)
 	ZEND_PARSE_PARAMETERS_END();
 
 	if (ZEND_NUM_ARGS() == 3) {
-		scale = (int) (scale_param < 0 ? 0 : scale_param);
+	    if (scale_param < 0 || scale_param > INT_MAX) {
```

Comment:
Please use tabs for indentation in C sources (but spaces in PHPTs).

Reply:
I think, there should be linter for this kind of things. So you don't have to point at this every new php dev, who wants to help his lang